src/test/java/Hmwk17.java

```diff
@@ -0,0 +1,86 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Hmwk17 extends BaseTest{
+
+
+@Test
+        public void newPlaylist() throws InterruptedException {
+
+        WebElement CreateNewPlaylist = driver.findElement(By.xpath("//*[@title= 'Create a new playlist']"));
+        CreateNewPlaylist.click();
+
+
+        WebElement SelectNewPlaylist = driver.findElement(By.xpath("//*[contains(text(),‘New Playlist’)]"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/199168640-30f81d00-589f-4faa-b1af-b39ae577dab0.png)

Fix it for this

 `WebElement SelectNewPlaylist = driver.findElement(By.xpath("//*[contains(text(),'New Playlist')]"));`